src/Microsoft.ML.Data/Transforms/ConcatTransform.cs

```diff
@@ -527,6 +527,19 @@ private static VersionInfo GetVersionInfo()
 
         public override ISchema Schema => _bindings;
 
+        public ConcatTransform(IHostEnvironment env, IDataView input, string outputColumn, params string[] inputColumns)
```

Comment:
>string outputColumn, params string[] inputColumns [](start = 70, length = 49)

This usage is inconsistent with your usage elsewhere, where the source is first, and the names of the output is second.

I would recommend actually making name of output first, and name of inputs second (as you have done here)... since I think the situation where you have multiple `params` to indicate input columns is more common. Indeed I cannot think of a single case where a `params` describes output columns, but I can think of instances where it describes input columns.

Reply:
I have renamed to "name" and "source" as well to be consistent with their naming in Argument object.

---
In reply to: [197985474](https://github.com/dotnet/machinelearning/pull/405#discussion_r197985474) [](ancestors = 197985474)